src/tasks/lint-locales.js

```diff
@@ -0,0 +1,74 @@
+const gulp = require('gulp');
+const plumber = require('gulp-plumber');
+const size = require('gulp-size');
+const _ = require('lodash');
+const registeredLinters = require('@shopify/theme-lint').linters;
+
+const config = require('./includes/config.js');
+const utils = require('./includes/utilities.js');
+const messages = require('./includes/messages.js');
+const Reporter = require('./includes/lint-reporter.js').default;
+
+/**
+ * Copies locales to the `/dist/locales` directory
+ *
+ * @param {String} directory
+ * @returns {Stream}
+ * @private
+ */
+function processLocales(directory) {
+  messages.logProcessFiles('build:locales');
+
+  return gulp.src(directory, {base: config.src.root})
+    .pipe(plumber(utils.errorHandler))
+    .pipe(size({
+      showFiles: true,
+      pretty: true,
+    }))
+    .pipe(gulp.dest(config.dist.root));
+}
+
+/**
+ * Recursively loops over all translations and keeps track of successes
+ * and failures. The reporter then outputs the results once completed.
+ *
+ * @param {String} directory
+ * @param {Array} linters
+ * @returns {Function|String} Recursive function call or finalized output
+ * @private
+ */
+function lintLocales(directory, linters, reporter = new Reporter()) {
+  const Linter = linters[0];
```

Comment:
At the moment we only define a single one, but `theme-lint` was build in such a way that we could easily add additional ones.

@chrisberthe would it be better to just name the task `lint` and run all the linters? We could probably move the `lint()` function from `cli.js` to `index.js` and export it there (https://github.com/Shopify/theme-lint/blob/b5755fe5ffe0c8ec04b3bafbb1c6e0e711d47a09/cli.js#L22-L37)